layout/paypal.ejs

```diff
@@ -0,0 +1,39 @@
+<form action="https://www.paypal.com/cgi-bin/webscr" method="post" target="paypal">
```

Comment:
1. hexo doesnt do anything like that for site generation
2. why do we even need it its just a form, there's no user flow to test. You click paypal a tab is opened and you stop.